src/System.Private.CoreLib/shared/System/Convert.cs

```diff
@@ -2518,21 +2520,127 @@ public static unsafe bool TryToBase64Chars(ReadOnlySpan<byte> bytes, Span<char>
                 charsWritten = ConvertToBase64Array(outChars, inData, 0, bytes.Length, insertLineBreaks);
                 return true;
             }
+        }      
+
+        internal static readonly Vector128<byte> s_base64ShuffleMask = Ssse3.IsSupported ? Vector128.Create((byte)
```

Comment:
Why `Ssse3.IsSupported`?

Reply:
@fiigii Ah, I see, Vector128.Create has a software fallback, will remove.

Reply:
Would it be better to use `default` when `Ssse3.IsSupported` results in `false`? So the software fallback isn't executed, as it isn't needed anyway.

Reply:
@gfoidl I guess the software fallback won't be executed anyway since the code that uses these fields will be eliminated if SSSE3 is not supported

Reply:
oh, actually no it will - Convert already has an implicit static cctor